.github/workflows/gradle.yml

```diff
@@ -34,14 +34,9 @@ jobs:
       - name: Build with Gradle
         uses: gradle/gradle-build-action@62cce3c597efd445cd71ee868887b8b1117703a7
         with:
-          arguments: core:shadowJar paper-support:shadowJar
+          arguments: shadowAll
```

Comment:
nie powinno być
`chatformatter-paper-plugin:shadowAll`
?

Reply:
Taska rejestrujesz globalnie, więc to bez różnicy